components/edit-collective/EditVirtualCardModal.tsx

```diff
@@ -102,6 +104,24 @@ const collectiveMembersQuery = gql`
   }
 `;
 
+export const virtualCardsAssignedToCollectiveQuery = gql`
+  query VirtualCardsAssignedToCollective($hostSlug: String!, $collectiveSlug: String!) {
+    host(slug: $hostSlug) {
+      id
+      allCards: hostedVirtualCards(collectiveAccountIds: [{ slug: $collectiveSlug }], status: [ACTIVE, INACTIVE]) {
+        totalCount
+      }
+      cardsMissingReceipts: hostedVirtualCards(
+        collectiveAccountIds: [{ slug: $collectiveSlug }]
+        status: [ACTIVE, INACTIVE]
+        hasMissingReceipts: true
+      ) {
+        totalCount
+      }
+    }
+  }
+`;
+
 const VirtualCardPoliciesQuery = gql`
   query VirtualCardPoliciesQuery($slug: String) {
```

Comment:
It's not new but the convention is that it should be `VirtualCardPolicies` and not `VirtualCardPoliciesQuery`

https://github.com/opencollective/opencollective/blob/main/rfcs/011-graphql.md